lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/HeartBeatManager.cs

```diff
@@ -84,32 +87,20 @@ public HeartBeatManager(EvaluatorSettings settings, IRemoteIdentifier remoteId)
             }
         }
 
-        [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1307:AccessibleFieldsMustBeginWithUpperCaseLetter", Justification = "Intended to be private, exposed now before using future injection")]
-        public EvaluatorRuntime _evaluatorRuntime { get; set; }
+        public EvaluatorRuntime Evaluatorruntime { get; set; }
```

Comment:
Shouldn't this be `EvaluatorRuntime`? Also, please document and consider make this `internal`.